packages/graphql/src/services/learningElements.ts

```diff
@@ -78,9 +79,9 @@ function evaluateQuestionResponse(
           }),
           percentile: pointsPercentage ?? 0,
         }
-      } else if (data.type === QuestionType.MC) {
+      } else if (questionData.type === QuestionType.MC) {
         const pointsPercentage = gradeQuestionMC({
-          responseCount: data.options.choices.length,
+          responseCount: questionData.options.choices.length,
           response: response.choices!,
```

Comment:
Is there a specific reason, why we still need to force TS to accept `response.choices` here as it seems to accept it for SC questions above? - same for KPRIM below

Reply:
changed to type casting. type narrowing does not work here, as response is on the instance, not within question data. maybe would work if we would narrow types on the question instance, but that will require prisma schema changes.